data/json/snippets/nightmares.json

```diff
@@ -43,11 +43,85 @@
+      },
+      {
+        "id": "nightmare_soldier2",
+        "text": "With another dissent into slumber, another nightmare comes whispering through your mind.\n\nA muddle of bodies, gyrating arms, and yelling faces shift before your vision, separated from you by the links of a chain fence strung with barbed wire.  Your hair flutters in the wind, tugged by the backwash of jet engines, as you find yourself sandwiched between a large transport plain painted in olive drab idling on an airstrip at your back and the yelling, fenced-away crowd at your front.  It's a scene you know all too well: what military brass would have termed a \"tactical retreat\" of \"priority assets\" from an \"untenable situation,\" in layman's terms: a military evacuation.\n\nSoldiers run up the aircraft's loading ramp as you yell for the assembly to keep their distance, half-heartedly razing your rifle as your pleas, lost under the plain's turbines, go unheeded by the pressing horde.  Your attention, however, is drawn further down the line when a sudden uproar flares up — somebody's breached the fence!  Swinging your weapon to face the mass of civilians dogpiling the breach, you hesitate as an elderly man with a shock of white hair pushes through the melee.nnEven in sleep, you grow nauseous when the senior pauses to lift a number of children through the gap before he clambers through himself, urging the young group towards the plain.\n\nHeedless of the fact that your rifle's laser sight is dancing rings about his forehead, you can barely hear the man bellowing over the tumult, \"YOU BASTARDS, YOU BASTARDS!  YOU'RE KILLING US ALL!  IF YOU'RE GOING TO RUN, TAKE THEM; TAKE THEM WITH YOU AT LEAST!\"\n\nYou close your eyes.  With 12 metres between the group and the plain, you had orders.\n\nThe sharp clack as you cock the rifle succeeds in bringing the last of the children to tears, though they keep coming.\n\n7 metres away, you feel the safety catch clack down below your thumb, and numbly perceive that the man's wearing spectacles mended with duct tape.  They still keep coming.\n\n4 metres, and your laser sight's painting a red splotch on the man's chest, inches above the curls of a small child clinging to his abdomen.  They don't stop coming.\n\n2 metres away...\n\nYour eyes burn as you startle awake, your complexion pallid.  The children's screams still reverberate in your head."
```

Comment:
This is a good example of what I was getting at above. I love what you are going for, but this is a *lot* of text for each nightmare, and this assumes a *LOT* about my character and who they are. These probably need to be a great deal shorter, with much less presumption about the player avatar experiencing them.